crypto/ocsp/test/aws/OCSP-TEST.md

```diff
@@ -27,9 +26,10 @@ The leaf cert/key. OCSP responses will be generated for this cert.
 ## OCSP response
 * ocsp_response.der
 
-DER formatted OCSP response for the Server Cert. 
+DER formatted OCSP response for the Server Cert.
 
 ## Generating a new OCSP response for the leaf cert
+Should not be necessary, our OCSP time stamps are hardcoded to avoid tests from expiring. The current response expires in 10 years.
```

Comment:
Nit: Strictly speaking, hard-coding a time stamp doesn't prevent certificates from expiring?

Reply:
Updated the bad wording in there :)